app/src/js/components/Requests/request.js

```diff
@@ -248,6 +270,46 @@ class RequestOverview extends React.Component {
     );
   }
 
+  AssignUWGmembers(record, filteredUsers) {
```

Comment:
No way to see or edit already assigned uwg members

Reply:
It is a multi select dropdown. once member is selected you will see their name in drop down and there is X next to each member. so it can be edited. I can give you a quick demo locally.

Reply:
Sorry I wasn't clear. There is no way to see or edit assigned uwg members after the initial assignment. i.e. once you send the API request or refresh the page.
You're right that you can remove the member from the dropdown before you submit the API request to assign the member.